app/models/location_polygon.rb

```diff
@@ -5,6 +5,7 @@ class LocationPolygon < ApplicationRecord
   scope :regions, -> { where(location_type: "regions") }
 
   def self.with_name(location)
+    location.strip!
```

Comment:
wouldnt the form model be a better place to do this?

Reply:
Yes it would